src/components/overview/StyleSelector/StyleSelector.jsx

```diff
@@ -67,7 +67,7 @@ const StyleSelector = ({state, updateState}) => {
                   }
                 })}
               </div>
-              <div id='styleBottom'>
+              <div id='StyleBottom'>
```

Comment:
Aren't CSS IDs supposed to be in camelCase?

Reply:
good catch!
just changed it thank you.